scripts/delete_dataset.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+import boto3
+import click
+import json
+import logging
+import os
+import sys
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from backend.corpora.common.corpora_config import CorporaDbConfig
+from backend.corpora.common.utils.json import CustomJSONEncoder
+from backend.corpora.common.utils.db_session import db_session_manager, DBSessionMaker
+from backend.corpora.common.entities.dataset import Dataset
+
+from urllib.parse import urlparse
+
+logging.basicConfig()
+logger = logging.getLogger(__name__)
+
+os.environ["CORPORA_LOCAL_DEV"] = "1"
+s3 = boto3.resource("s3")
+
+
+@click.command()
+@click.argument("uuid")
+@click.option(
+    "--deployment", default=os.getenv("DEPLOYMENT_STAGE", "test"), help="The name of the deployment to target."
+)
```

Comment:
Maybe state the default in the help text

Reply:
add the default state